src/codegen/ast_interpreter.cpp

```diff
@@ -1144,9 +1163,17 @@ Box* astInterpretFrom(CompiledFunction* cf, AST_expr* after_expr, AST_stmt* encl
 
     for (const auto& p : locals->d) {
         assert(p.first->cls == str_cls);
-        auto name = static_cast<BoxedString*>(p.first)->s;
-        InternedString interned = cf->clfunc->source->getInternedStrings().get(name);
-        interpreter.addSymbol(interned, p.second, false);
+        std::string name = static_cast<BoxedString*>(p.first)->s;
```

Comment:
minor nit -- this will make a copy (I guess the original version did too), could you have this be a `const std::string&`?